app.R

```diff
@@ -0,0 +1,237 @@
+library(solvebio)
+library(data.table)
+login(api_key="af2eab3c7aeda65b50f400ca941c0296be812683", api_host="https://api.solvebio.com")
```

Comment:
this is not in a file from this pull request, I deleted that file before pushing, so it may be from an older commit. there's just the SwissProt rmd file and that doesn't have the api key